content/user/advanced-modules/Cases with Portal.adoc

```diff
@@ -39,9 +39,16 @@ automatically generate cases can be found link:../../../admin/administration-pan
 
 == Downloading
 
-The plug-in can be downloaded for free from the SuiteCRM site at 
+The plug-in can be downloaded for free from the SuiteCRM site at
 the https://suitecrm.com/resources/suitecrm-joomla-portal-plug-in/[Joomla Portal Plug-in page^].
 
+== Generate Public & Private Keys
+
+Before beginning the setup for Joomla in your local SuiteCRM instance, you should generate your public and private key
+for the V8 API. This can be done here:
```

Comment:
The Joomla portal currently still uses the V4 API so not sure if this is specifically required for that